evdns.c

```diff
@@ -3481,6 +4110,37 @@ str_matches_option(const char *s1, const char *optionname)
 		return 0;
 }
 
+/* exported function */
+int
+evdns_server_port_set_option(struct evdns_server_port *port,
```

Comment:
let's make `option` enum and value `size_t` (can be done `void *` but I don't think it is a good idea, will add separate function for strings if any, since instead of string we can also accept some enum)

Reply:
Done.